src/pages/markets/static/content/_synthetic.tsx

```diff
@@ -8,33 +8,34 @@ import SevenTrading from 'images/svg/jump-indices/jump-indices-24-7-trading.svg'
 import PractiseIcon from 'images/svg/markets/aim-new.svg'
 import TradeIcon from 'images/svg/markets/trade-new.svg'
 import WithdrawIcon from 'images/svg/markets/deposit-and-withdrawal-new.svg'
+import { TString } from 'types/generics'
+
+export type SimpleStepContentElement = {
+    header: TString
+    text: TString
+    icon: ReactElement
+}
 
 type Synthetic = {
     src: string
     text: React.ReactNode
     alt: string
 }
 
-export const simple_step_content_synthetic = [
+export const simple_step_content_synthetic: SimpleStepContentElement[] = [
     {
-        header: <Localize translate_text="Practise" />,
-        text: (
-            <Localize translate_text="Open a demo account and practise with an unlimited amount of virtual funds." />
-        ),
+        header: '_t_Practise_t_',
+        text: '_t_Open a demo account and practise with an unlimited amount of virtual funds._t_',
         icon: <img src={PractiseIcon} alt="Practise" width="32" height="32" />,
     },
     {
-        header: <Localize translate_text="Trade" />,
-        text: (
-            <Localize translate_text="Open a real account, make a deposit, and start trading synthetics and other markets." />
-        ),
+        header: '_t_Trade_t_',
+        text: '_t_Open a real account, make a deposit, and start trading synthetics and other markets._t_',
         icon: <img src={TradeIcon} alt="Trade" width="32" height="28" />,
     },
     {
-        header: <Localize translate_text="Withdraw" />,
-        text: (
-            <Localize translate_text="Conveniently withdraw your funds through any of our supported withdrawal methods." />
-        ),
+        header: '_t_Withdraw_t_',
+        text: '_t_Conveniently withdraw your funds through any of our supported withdrawal methods._t_',
         icon: <img src={WithdrawIcon} alt="Withdraw" width="32" height="32" />,
     },
 ]
```

Comment:
why we don't refactor array `Syntetic` below? 🥲

Reply:
I was refactoring custom components. I checked with @niloofar-deriv and she asked to refactor only custom component usage part

Reply:
Clear, thanks 👍